powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/request/AssertionVerificationRequest.java

```diff
@@ -52,5 +52,6 @@ public class AssertionVerificationRequest {
     private List<String> allowedTopOrigins = new ArrayList<>();
     private boolean requiresUserVerification;
     private String expectedChallenge;
+    private boolean visualSignature;
```

Comment:
Can we unify the naming? Sometimes `visualSignature` is used and sometimes `isVisualSignature` is used.